cmd/erasure-errors.go

```diff
@@ -20,10 +20,10 @@ package cmd
 import "errors"
 
 // errErasureReadQuorum - did not meet read quorum.
-var errErasureReadQuorum = errors.New("Read failed. Insufficient number of drives online")
+var errErasureReadQuorum = errors.New("Read failed. Insufficient number of quorum drives")
```

Comment:
Not sure what a "quorum drive" is. TBH I prefer the old wording, or
```suggestion
var errErasureReadQuorum = errors.New("Read failed. Insufficient number of drives online for quorum")
```

Reply:
I kept the old wording